src/main/resources/db/migration/VS20230515_52__IJPS-52_statusHistory.sql

```diff
@@ -0,0 +1,10 @@
+
+CREATE TABLE public.status_history (
+	status_history_ID bigint not null,
+  sitting_record_ID bigint not null,
+  status_ID varchar(255) not null,
+  change_date_time timestamp not null,
+  change_by_user_ID status_id varchar(255) not null,
+  CONSTRAINT pk_status_history PRIMARY KEY (status_history_id),
+	CONSTRAINT fk_status_history FOREIGN KEY (sittingRecord.sitting_record_id)
```

Comment:
does this foreign link work? the table is sitting_record I think it might need to be 
```
CONSTRAINT fk_status_history FOREIGN KEY  (sitting_record_id) REFERENCES 
 public.sitting_record(sitting_record_id);
```